seismicpro/src/refractor_velocity/utils.py

```diff
@@ -28,3 +31,49 @@ def postprocess_params(params):
     if is_1d:
         return params[0]
     return params
+
+def dump_refractor_velocities(refractor_velocities, path, encoding="UTF-8"):
+    """Dump the parameters of passed velocity models to a file.
+
+    Parameters
+    ----------
+    refractor_velocities : RefractorVelocity or iterable of RefractorVelocity.
+        Refractor Velocity instances to dump to the file.
+    path : str
+        Path to the created file.
+    encoding : str, optional, defaults to "UTF-8"
+        File encoding.
+    """
+    rv_list = to_list(refractor_velocities)
+    columns = ['name_x', 'name_y', 'x', 'y'] + list(rv_list[0].params.keys())
+    data = np.empty((len(rv_list), len(columns)), dtype=object)
+    for i, rv in enumerate(rv_list):
+        data[i] = [*rv.coords.names] + [*rv.coords.coords] + list(rv.params.values())
+    df =  pd.DataFrame(data, columns=columns).convert_dtypes(convert_floating=False)
+    df.to_string(buf=path, float_format="%.2f", index=False, encoding=encoding)
```

Comment:
Let's pass formatter as function, as they ask in docs. Then pd.float32 are formatted as expected.

```suggestion
    df =  pd.DataFrame(data, columns=columns).convert_dtypes()
    df.to_string(path, index=False, encoding=encoding, float_format=lambda x: "{:.2f}".format(x))
```